src/utils/index.ts

```diff
@@ -3,6 +3,18 @@ import config from "../config";
 import { awesomenessInterval } from "./values";
 import dynamoDB from "../services/db";
 import { createTeammateTable } from "../services/db/createTable";
+import { BeastieLogger } from "./Logging";
+
+function CallTwitchApi(uri, options) {

```

Comment:
Could this function's first letter be lower cased, as with other functions? There's no style guide, but to me it makes sense to keep functions and methods lowerCamelCase, unless they represent a class. :P ```suggestion
function callTwitchApi(uri, options) {

```